pandas/core/arrays/datetimelike.py

```diff
@@ -791,6 +795,10 @@ def __sub__(self, other):
             else:  # pragma: no cover
                 return NotImplemented
 
+            if is_timedelta64_dtype(result) and isinstance(result, np.ndarray):
```

Comment:
worth comments here?

Reply:
Not really.  It would just say `# wrap result in TimedeltaArray` which is pretty redundant given the one-liner here